File: Arjuna/versions/PR2.0_Alpha_2/Tests/Local/UidUniqueness/UidUniqueness.cc
/*
 * Copyright (C) 1993
 * 
 * Department of Computing Science,
 * The University,
 * Newcastle upon Tyne,
 * UK.
 */

/*
 * $Id: UidUniqueness.cc,v 1.1 1993/11/03 15:01:43 nmcl Exp $
 */

#include <System/iostream.h>
#include <Common/Uid.h>

int main(int, char*[])
{
    Uid uids[16];

    for (int i = 0; i < 16; i++)
	cout << uids[i] << '\n' << flush;
}

File: JTSArjuna/JTSArjuna-1.2.4/JavaGandiva/tests/Uid/UidUniqueness.java
/*
 * Copyright (C) 1996, 1997, 1998,
 *
 * Department of Computing Science,
 * University of Newcastle upon Tyne,
 * Newcastle upon Tyne,
 * UK.
 *
 * $Id: UidUniqueness.java,v 1.1 1998/11/18 09:01:38 nmcl Exp $
 */

import com.arjuna.JavaGandiva.Common.*;

public class UidUniqueness
{
    
public static void main (String[] args)
    {
	for (int i = 0; i < 100; i++)
	{
	    Uid u = new Uid();
	    System.out.println(u + " " + u.hashCode());
	}
    }
    
};
